input/pagecontent/RAD-Y1.md

```diff
@@ -140,31 +139,33 @@ In addition to the common set above, there are a number of useful optional attri
-
-> In other words, the Sender shall not presume that the Receiver can download and render the linked content.
+For all rendered reports, the Sender shall set the presentedForm.contentType with a value corresponding to the rendered report format.
 
 The Sender shall encode the rendered report that is referenced by DiagnosticReport.presentedForm in one of the following ways:
 - Encode the rendered report as a base64Binary in DiagnosticReport.presentedForm.data
+- Host the rendered report somewhere and provide the url in DiagnosticReport.presentedForm.url.
 - Encode the rendered report as a base64Binary in a [Binary Resource](https://www.hl7.org/fhir/binary.html) and this Binary Resource is referenced in DiagnosticReport.presentedForm.url.
   - The Binary Resource shall be in the Bundle. See FHIR Resolving references in Bundles at [http://hl7.org/fhir/bundle.html#references](http://hl7.org/fhir/bundle.html#references). 
-- Host the rendered report somewhere and provide the url in DiagnosticReport.presentedForm.url.
 
-The Sender shall populate accurate .hash and .size for the rendered report content in presentedForm: 
-* Where the presentedForm is a Binary Resource instance, the .hash and .size measure the raw artifact that has been base64encoded in the Binary.data element.  
-* Where the presentedForm is hosted elsewhere, not as a Binary Resource, the .hash and the .size shall represent the rendered report content that would be retrieved using the mime-type specified in contentType element. 
+The Sender shall populate accurate values for hash and size for the rendered report content in DiagnosticReport.presentedForm.hash and DiagnosticReport.presentedForm.size: 
+* Where the presentedForm is a Binary Resource instance, the .hash and .size shall represent the raw artifact that has been base64encoded in the Binary.data element.  
+* Where the presentedForm is hosted elsewhere, not as a Binary Resource, the .hash and the .size shall represent the rendered report content that would be retrieved using the mime-type specified in DiagnosticReport.presentedForm.contentType. 
 
-For all rendered reports, the Sender shall set the presentedForm.contentType with a value corresponding to the rendered report format.
+A Sender that supports the **PDF Report Option**, if configured, shall also attach a semantically equivalent diagnostic report in PDF format in the DiagnosticReport.presentedForm attribute. The presentedForm.contentType shall have the value "application/pdf". The Sender shall include in the PDF report all text and linked contents as in the HTML report. The Sender may preserve the linked contents as hyperlinks, or substitute the linked contents with the actual rendered contents and embedded them in the PDF file.
+
+###### TO DO: Find the right home for this
```

Comment:
The followings should be part of the rendered report requirement for the Sender.